### PR TITLE
St robert pavlov lesson 4

### PR DESCRIPTION
**Pull request by roberts-pavlovs**

### CODE REVIEW

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_1.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package main.java.students.student_robert_pavlov.homework.lesson_4.Level_1.Level_1;
+
+public class Lesson_4_Level_1_Task_1 {
+    public static void main(String[] args) {
+
+        int number = 15;
+        if (number > 50) {
+            System.out.println("TRUE");
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_1.java:8]: Output message does not contain any meaningful for the user info. Please fix.

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_2.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        } else if (number != 15) {
+            System.out.println("Number is 15");
+        } else if (number > 10 && number == 15) {
+            System.out.println("Number is bigger than ten and is equals 15");
+        } else {
+            System.out.println("Who are you?");
+        }
+
+    }
+}
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_2.java:18]: This is not what was required in Lesson_4 Task_2. Please fix.

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_3.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            System.out.println("Tuesday");
+        } else if (day == 3) {
+            System.out.println("Wednesday");
+        } else if (day == 4) {
+            System.out.println ("Thursday");
+        } else if (day == 5) {
+            System.out.println("Friday");
+        } else if (day == 6) {
+            System.out.println("Saturday");
+        } else if (day == 8) {
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_3.java:20]: `day == 8` --> `day == 7`

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_4.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        Scanner input = new Scanner(System.in);
+        number = input.nextInt();
+
+        if (number % 2 == 0){
+        System.out.println(" even number ");}
+
+         else {
+            System.out.println(" the number is not even ");}
+
+    }
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_4.java:20]: Please fix formatting.

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            System.out.print("Input second number: ");
+        int y = in.nextInt();
+            System.out.print("Input third number: ");
+        int z = in.nextInt();
+
+        if (x == y && x == z)
+        {
+            System.out.println("All numbers are equal");
+        }
+        else if ((x == y) || (x == z) || (z == y))
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java:21]: Why do we need this `else if` statement if this statement is not reachable?

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Please check comments.

P.S. Please include in PR changes only related to the level. For example you named this branch level-4 but it contains homework from lesson 1, lesson 2 and lesson 3.

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_4.java (hunk 4 above)
Comment (roberts-pavlovs) [reply src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_4.java:20]: Can't find error

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java (hunk 5 above)
Comment (roberts-pavlovs) [reply src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java:21]: We don't need it, fixed the bug.

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_4.java (hunk 4 above)
Comment (antdolg1) [reply src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_1/Lesson_4_Level_1_Task_4.java:20]: Just press `Ctrl+Alt+L` and Intellij IDEA will apply auto format.

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/Level_1/Lesson_4_Level_1_Task_3.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
+package main.java.students.student_robert_pavlov.homework.lesson_3.Level_1;
+
+public class Lesson_4_Level_1_Task_3 {
+    public static void main(String[] args) {
+
+        int day = 7;
+
+        if (day == 5) {
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_3/Level_1/Lesson_4_Level_1_Task_3.java:8]: `if (day == 5)` --> `if (day == 1)`

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/Level_1/Lesson_4_Level_1_Task_4.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+        number = input.nextInt();
+
+        if (number % 2 == 0){
+             System.out.println(" even number ");}
+
+         else {
+             System.out.println(" the number is not even ");}
+
+    }
+}
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_3/Level_1/Lesson_4_Level_1_Task_4.java:21]: Please auto format the code `Ctrl+Alt+L`

File: src/main/java/students/student_robert_pavlov/homework/lesson_3/Level_1/Lesson_4_Level_1_Task_3.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
+package main.java.students.student_robert_pavlov.homework.lesson_3.Level_1;
+
+public class Lesson_4_Level_1_Task_3 {
+    public static void main(String[] args) {
+
+        int day = 7;
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_3/Level_1/Lesson_4_Level_1_Task_3.java:6]: We should read this value from user input via scanner.

File: src/main/java/teacher/students/student_robert_pavlov/lesson_2/PrimitiveVariables.java [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+        boolean myBoolean = true;
+                System.out.println("myBoolean value is: " + myBoolean);
+
+        char myChar = 'b';
+                System.out.println("myChar value is: " + myChar);
+
+
+    }
+}
```
Comment (antdolg1) [inline src/main/java/teacher/students/student_robert_pavlov/lesson_2/PrimitiveVariables.java:32]: Please remove this file from teacher folder.

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+        System.out.print("Input first number: ");
+        int x = in.nextInt();
+        System.out.print("Input second number: ");
+        int y = in.nextInt();
+        System.out.print("Input third number: ");
+        int z = in.nextInt();
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java:15]: In Lesson 4 Level 2 Task 7 it is required from user to input only 2 numbers, not 3. Please fix according to task requirements.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Some issues are fixed, some not. Please check new comments.

File: src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java [hunk 11]
```diff
@@ -7,20 +7,19 @@ public class Lesson_4_Level_2_Task_7 {
...
-        if (x == y && x == z) {
-            System.out.println("All numbers are equal");
-        } else if ((x == y) || (x == z) || (z == y)) {
+        System.out.print(" Input second number: ");
+        int y = in.nextInt();
 
-        } else {
+        if (x == y) {
+            System.out.println(" All numbers are equal ");
+        } else if ((x != y)) {
```
Comment (antdolg1) [inline src/main/java/students/student_robert_pavlov/homework/lesson_4/Level_1/Level_2/Lesson_4_Level_2_Task_7.java:19]: 1. No need for double `(( ))`
2. Actually no need for this check at all, you can safely replace it with `else`

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**